packages/react-dev-utils/FlowTypecheckPlugin.js

```diff
@@ -109,6 +108,7 @@ function getFlowVersion(options) {
 }
 
 function initializeFlow(projectPath, flowconfig, otherFlowTypedDefs) {
+  const flowTypedPath = path.join(projectPath, 'node_modules', '.bin', 'flow-typed');
```

Comment:
So yarn doesn't pass with this and the corresponding npm version for node 4 doesn't either